scripts/Engineering/EnginX.py

```diff
@@ -578,14 +838,22 @@ def focus_texture_mode(run_number, van_curves, van_int, focus_directory, focus_g
         for row in group_reader:
             banks.update({row[0]: ','.join(row[1:])})
 
-    # loop through the banks described in the csv, focusing and saing them out
+    # loop through the banks described in the csv, focusing and saving them out
     for bank in banks:
-        output_ws = "engg_focusing_output_ws_texture_bank_{}"
-        output_ws = output_ws.format(bank)
-        simple.EnggFocus(InputWorkspace=ws_to_focus, OutputWorkspace=output_ws,
-                         VanIntegrationWorkspace=van_integrated_ws, VanCurvesWorkspace=van_curves_ws,
-                         SpectrumNumbers=banks.get(bank))
-        _save_out(run_number, focus_directory, focus_general, output_ws, "ENGINX_{}_texture_{}{{}}", bank)
+        sample_ws_clone = simple.CloneWorkspace(ws_to_focus)
```

Comment:
Do we need to clone these workspaces if we apply the sensitivity correction outside the loop over banks?